README.md

```diff
@@ -20,6 +20,7 @@ Usage:
 Flags:
       --anonymize                    flag to anonymize collected NSX resources (default false)
       --color                        flag to enable color output (default false)
+      --synth-create-dns-policy     flag to create a policy allowing access to target env dns pod (default true)
```

Comment:
move to the correct place in the list of flags

Reply:
fixed